pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[project]
+name = "mimir-worker-k8s"
+version = "0.0"  # this is in fact irrelevant
+requires-python = ">=3.12"
+
+dependencies = [
+    "ops>=1.5.0",
+    # Charm relation interfaces
+    "pydantic",
+    
+    # ---PYDEPS---
+    "cosl>=0.0.43",
+    # lib/charms/tempo_coordinator_k8s/v0/charm_tracing.py
+    "opentelemetry-exporter-otlp-proto-http==1.21.0",
+]
+
+[project.optional-dependencies]
+lint = [
+    "ruff",
+    "codespell",
+]
+fmt = ["ruff"]
+static = ["pyright"]
+unit = [
+    "pytest",
+    "coverage[toml]",
+]
+scenario = []
```

Comment:
I vote we combine these into one unit testing group.  I don't know why these would ever depart.